docs/reference/indices/simulate-template.asciidoc

```diff
@@ -0,0 +1,308 @@
+      }
+    },
+    "aliases": {
+      "mydata": { }
+    }
+  },
+  "priority": 10,
+  "composed_of": ["component_template1", "other_component_template"],
+  "version": 3,
+  "_meta": {
+    "description": "my custom"
+  }
+}
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+DELETE _index_template/*
+DELETE _component_template/*
+--------------------------------------------------
+// TEARDOWN
+////
+
+[source,console]
+--------------------------------------------------
+POST /_index_template/_simulate/template_1
+--------------------------------------------------
+
+[[simulate-template-api-request]]
+==== {api-request-title}
+
+`POST /_index_template/_simulate/<index-template>`
+
+
+[[simulate-template-api-path-params]]
+==== {api-path-parms-title}
+
+`<index-template>`::
+(Optional, string)
+The name of the index template to simulate.
+To simulate an arbitrary template configuration, 
+omit this parameter and specify the template configuration in the request body.
+
+[[simulate-template-api-query-params]]
+==== {api-query-parms-title}
+
+`cause`::
+(Optional, string) The reason for using the specified template for the simulation. 
+
+`create`::
+(Optional, boolean) If `true`, the template passed in the body is 
+only used if no existing template matches the same index pattern. 
+If `false`, the template with the highest priority is used for the simulation.
+Note that the template is not permanently added or updated in either case, 
+it is only used for the simulation.
+Defaults to `false`.
+
+include::{es-ref-dir}/rest-api/common-parms.asciidoc[tag=master-timeout]
+
+[[simulate-template-api-request-body]]
+==== {api-request-body-title}
+
+`composed_of`::
+(Optional, array of strings)
+An ordered list of component template names. Component templates are merged in the order
+specified, meaning that the last component template specified has the highest precedence. See 
+<<multiple-component-templates,Composing multiple component templates>> for an example.
+
+`index_patterns`::
+(Required, array of strings)
+Array of wildcard expressions
+used to match the names of indices during creation.
+
+`priority`::
+(Optional, integer)
+Priority to determine index template precedence when a new index is created. The index template with
+the highest priority is chosen. If no priority is specified the template is treated as though it is
+of priority 0 (lowest priority).
+This number is not automatically generated by {es}.
+
+`template`::
+(Optional, object)
+This is the template to be applied, may optionally include a `mappings`,
+`settings`, or `aliases` configuration.
+
+`version`::
+(Optional, integer)
+Version number used to manage index templates externally.
+This number is not automatically generated by {es}.
+
+`_meta`::
+(Optional, object)
+Optional user metadata about the index template. May have any contents.
+This map is not automatically generated by {es}.
```

Comment:
It may be good to include some suggestions here. Or mention that the data is arbitrary.
```suggestion
`_meta`::
(Optional, object)
Metadata about the index template.
This metadata is not used or automatically generated by {es}.
```